Improve graphql client bbes (#3825)
ballerina-platform/ballerina-distribution: examples/graphql-client-error-handling/graphql_client_error_handling.md

# GraphQL client - Handle error response

> The `execute` method of `graphql:Client` can fail and return a `graphql:ClientError`. The `graphql:ClientError` has different subtypes that can be handled differently based on the use case. Use the subtypes of `graphql:ClientError` to handle different types of errors differently.

Reviewer: ```suggestion
The `execute` method of the `graphql:Client` can fail and return a `graphql:ClientError`. The `graphql:ClientError` has different subtypes that can be handled differently based on the use case. Use the subtypes of the `graphql:ClientError` to handle different types of errors differently.
```

> The `execute` method of the `graphql:Client` can fail and return a `graphql:ClientError`. The `graphql:ClientError` has different subtypes that can be handled differently based on the use case. Use the subtypes of the `graphql:ClientError` to handle different types of errors differently.

Reviewer 2: ```suggestion
The Ballerina GraphQL module allows handling different errors occurred when executing the `execute` method of the `graphql:Client`. It returns a `graphql:ClientError` error which has different subtypes that can be handled differently based on the use case. Use the subtypes of the `graphql:ClientError` to handle different types of errors based on the use case.
```

> The `graphql:Client` allows handling different errors occurred when executing the `execute` method. It returns a `graphql:ClientError` error which has different subtypes that can be handled differently based on the use case. Use the subtypes of the `graphql:ClientError` to handle different types of errors based on the use case.

Reviewer: ```suggestion
The `graphql:Client` allows handling different errors occurred when executing the `execute` method. It returns a `graphql:ClientError` error, which has different subtypes that can be handled differently based on the use case. Use the subtypes of the `graphql:ClientError` to handle different types of errors based on the use case.
```
